### PR TITLE
read inputbox between espresso

### PR DESCRIPTION
**Pull request by ZzzzHui**
This hasn't been tested yet.
From what I see, Cappuccino uses Sepolia as L1. So tomorrow I will test with Sepolia

### CODE REVIEW

**Comment by fabiooshiro** [comment]
@ZzzzHui I added a error check and just return the error

File: internal/sequencers/espresso/read_input_box.go
Comment (fabiooshiro) [inline internal/sequencers/espresso/read_input_box.go:1]: Hey, how about changing the file name from read_input_box.go to input_box_reader.go? I think it fits better with our naming style.

Comment (ZzzzHui) [reply internal/sequencers/espresso/read_input_box.go:1]: Yes. Will do

**Comment by ZzzzHui** [comment]
I've done some tests on Sepolia, the codes are working as expected. The command to run is
```
./nonodo \
    --sequencer=espresso --from-block=1680465 --namespace=15343 --enable-debug \
    --contracts-application-address 0x590F92fEa8df163fFF2d7Df266364De7CE8F9E16 \
    --contracts-input-box-address 0x58Df21fE097d4bE5dCf61e01d9ea3f6B81c2E1dB \
    --contracts-input-box-block 6099597 \
    --rpc-url https://sepolia.infura.io/v3/<API_KEY>
```

We don't actually need the `input-box-block` because we start reading past inputs from the `l1_finalized_block` specified in the espresso's `--from-block`'s header

**Comment by fabiooshiro** [comment]
Hi @ZzzzHui, the build is broken.

**Review by fabiooshiro — APPROVED** [review]

**Pull request merged**